Makefile

```diff
@@ -94,3 +94,17 @@ rebuild-pipeline: ./local-test-repo/.git
 .PHONY: bench
 bench:
 	@cd ./local-test-repo/ && make -s bench
+
+.PHONY: start-prometheus-alertmanager
+start-prometheus-alertmanager:
+	cd ./prometheus/ && \
+	(test -f ../environments/production.env && \
+	 docker-compose --env-file=../environments/production.env up --detach) || \
```

Comment:
perhaps we can only have it running for production? we don't really care about having it run for dev.